srcpkgs/libnma/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'libnma'
+pkgname=libnma
+version=1.8.30
+revision=1
+build_style=meson
+build_helper="gir"
+hostmakedepends="cmake gettext glib-devel gtk-doc pkg-config vala"
+makedepends="gcr-devel gtk+3-devel iso-codes
+ mobile-broadband-provider-info NetworkManager-devel"
+depends="NetworkManager iso-codes"
+short_desc="NetworkManager panel applet for GNOME - runtime library"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://gitlab.gnome.org/GNOME/libnma"
+distfiles="https://gitlab.gnome.org/GNOME/libnma/-/archive/${version}/libnma-${version}.tar.gz"
+checksum=c874d83dd59468b68c79ff20e8f5a689a009c89b618e89b5939fbdc81776fbf5
+
+# Package build options
+build_options="gir"
+build_options_default="gir"
```

Comment:
Typically we only specify options when something about a package is actually optional. In this case it always needs gir. Good rule of thumb is that a build option should actually change the behavior of the build, ejemplos gratis: dependencies, arguments passed to the build system.

Reply:
Gotcha, I've removed those lines.